lib/matplotlib/tests/test_mathtext.py

```diff
@@ -93,7 +93,6 @@
     r'$\operatorname{cos} x$',  # github issue #553
     r'$\sum _{\genfrac{}{}{0}{}{0\leq i\leq m}{0<j<n}}P\left(i,j\right)$',
     r"$\left\Vert a \right\Vert \left\vert b \right\vert \left| a \right| \left\| b\right\| \Vert a \Vert \vert b \vert$",
-    r'$\mathring{A}  \stackrel{\circ}{A}  \AA$',
```

Comment:
Can we leave in the test as  `r'$\mathring{A}  \AA$',` ?  I don't think we other wise have coverage of those symbols and it will avoid the re-numbering of the tests which is very confusing.

If we really want to drop this test, we should leave a `None` in place and filter it a bit farther down just before we feed paramaterize.